GG/GG/Font.h

```diff
@@ -206,6 +209,22 @@ class GG_API Font
 
         virtual ~TextElement(); ///< Virtual dtor.
 
+        /** Rebind to \p whole_text, by binding the SubString data member text to \p whole_text.
+
+            Binding to a new \p whole_text is very fast compared to re-parsing the entire \p
+            whole_text and allows TextElements of TextElementType TEXT to be changed quickly if it
+            is known that the parse would be the same.
+
+            One use case is copying or copy constructing a TextControl.
+
+            Another use case is constructing a string and TextElement vector pair with
```

Comment:
I've made the following changes:
- I used your words from above to describe use cases
- changed the word rebind to attach
- added clarification to the `Substring` description that binding points the iterators into the new string.